faq.adoc

```diff
@@ -11,13 +11,15 @@ Quarkus is an Open Source project licensed under the https://www.apache.org/lice
 
 == Where can I get it?
 
-Quarkus is published in Maven Central, check out link:/extensions[which extensions] you need and just import them in your `pom.xml` to get Quarkus. We recommend you start your Quarkus experience via our link:/get-started[Getting Started guides].
+Quarkus is published in Maven Central. We recommend you start your Quarkus experience via our link:/get-started[Getting Started guides] or by downloading a scaffolded application from http://code.quarkus.io[code.quarkus.io].
 
 == Quarkus is stable?
 
-Yes, we consider Quarkus stable. Remember 95% of the features Quarkus apps use are provided by the ecosystem like Hibernate ORM, Eclipse Vert.x, Netty, RESTEasy, etc. These libraries are rock solid :)
+Yes, we consider Quarkus stable.
+Quarkus is used in production by a https://quarkus.io/blog/tag/user-story/[range of organisations].
+Remember 95% of the features Quarkus apps use are provided by the ecosystem like Hibernate ORM, Eclipse Vert.x, Netty, RESTEasy, etc. These libraries are rock solid :)
```

Comment:
I'm not sure about this content now tbh but probably not something to discuss in this PR.

I wonder if I would have `wide range of organizations (see some link:user stories)` because otherwise it's a bit limiting. But not really sure and also don't want to start a long discussion here.

Reply:
Yeah, I decided to go for "make it better than it currently is" rather than "make it optimal". I agree a bigger rewrite is needed.

Reply:
I don't have strong feelings about either wording, so I've gone with your suggestion and squashed.